toto/runlib.py

```diff
@@ -169,9 +165,8 @@ def create_link_metadata(name, materials, products, byproducts,
 
 def run_link(name, materials, products, toto_cmd_args, record_byproducts=False):
   """Performs all actions associated with toto run-link.
-  XXX: This should probably be atomic, i.e. all or nothing"""
+  XXX LP: This should probably be atomic, i.e. all or nothing"""
```

Comment:
Good thought. I must have adapted this from the `Seattle` codebase. Let's discuss this with regard to our [style guide lines](https://github.com/secure-systems-lab/code-style-guidelines).